clients/azuredevopsrepo/repo.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package azuredevops
+package azuredevopsrepo
```

Comment:
Does it matter that this was dead code before?

Reply:
In a technical sense, no, as parts of the codebase are as a library. So a dev could have written code which depends on "github.com/ossf/scorecard/v5/clients/azuredevops", and renaming this would be a breaking change for them.

But the `azuredevops` package wasn't merged before the v5.0.0 release, so the only way someone could have depended on this was with a [pseudo-version](https://go.dev/ref/mod#pseudo-versions). I'm not sure what the official stance is on pseudo-versions and semver, but I don't consider this breaking anymore.